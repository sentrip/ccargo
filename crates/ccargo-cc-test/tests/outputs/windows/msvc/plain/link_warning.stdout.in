warn.c
main.c
Generating Code...
   Creating library build\main.lib and object build\main.exp
LINK : warning LNK4217: symbol 'func' defined in 'warn.obj' is imported by 'main.obj' in function 'main'
