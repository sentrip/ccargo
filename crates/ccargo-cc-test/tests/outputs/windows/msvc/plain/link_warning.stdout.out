warning LNK4217: symbol 'func' defined in 'warn.obj' is imported by 'main.obj' in function 'main'
   --> LINK 
